### ISSUE TITLE
vmware_workstation acceptance tests

### ISSUE THREAD

**Issue by jantman** [jantman/vagrant-r10k#20]
I've gotten access to VMWare Workstation itself, and gotten a license for the plugin specifically for testing vagrant-r10k. However, I haven't been able to get VMW Workstation to run on my desktop (Arch Linux with 4.1.2-2 kernel). I have it running on my laptop, but it doesn't work with Vagrant; even with the latest plugin (`vagrant-vmware-workstation (3.2.11)` and Vagrant 1.6.5). The closest I've been able to get is waiting for the machine to boot and then "The DHCP leases file for the vmnet device 'vmnet8' could not be found. [...]". This appears to be the same as [Vagrant issue 6101](https://github.com/mitchellh/vagrant/issues/6101) that says to reinstall Workstation.

So, bottom line, I don't have any working VMWare Workstation acceptance tests yet, and it seems that [vagrant-spec](https://github.com/mitchellh/vagrant-spec/) as a whole is pretty unpolished. If more people start using Workstation, or opening issues specific to it, I'll revisit this. Until then, let's just do acceptance for Virtualbox.

WIP for this is in the issues/12final_vmware branch.
